src/pages/settings/Subscription/RequestEarlyCancellationPage/index.tsx

```diff
@@ -0,0 +1,139 @@
+        () => (
+            <Text>
+                {translate('subscription.requestEarlyCancellation.acknowledgement.part1')}
+                <TextLink href={CONST.TERMS_URL}>{translate('subscription.requestEarlyCancellation.acknowledgement.link')}</TextLink>
+                {translate('subscription.requestEarlyCancellation.acknowledgement.part2')}
+            </Text>
+        ),
+        [translate],
+    );
+
+    const manualCancellationContent = useMemo(
+        () => (
+            <View style={[styles.flexGrow1, styles.justifyContentBetween, styles.mh5]}>
+                <View>
+                    <Text style={styles.textHeadline}>{translate('subscription.requestEarlyCancellation.requestSubmitted.title')}</Text>
+                    <Text style={[styles.mt1, styles.textNormalThemeText]}>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part1')}
+                        <TextLink onPress={() => Report.navigateToConciergeChat()}>{translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.link')}</TextLink>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part2')}
+                    </Text>
+                </View>
+                <FixedFooter style={styles.ph0}>
+                    <Button
+                        success
+                        text={translate('common.done')}
+                        onPress={() => Navigation.goBack()}
+                        large
+                    />
+                </FixedFooter>
+            </View>
+        ),
+        [styles, translate],
+    );
+
+    const automaticCancellationContent = useMemo(
+        () => (
+            <View style={[styles.flexGrow1, styles.justifyContentBetween, styles.mh5]}>
+                <View>
+                    <Text style={styles.textHeadline}>{translate('subscription.requestEarlyCancellation.subscriptionCanceled.title')}</Text>
+                    <Text style={[styles.mt1, styles.textNormalThemeText]}>{translate('subscription.requestEarlyCancellation.subscriptionCanceled.subtitle')}</Text>
+                    <Text style={[styles.mv4, styles.textNormalThemeText]}>{translate('subscription.requestEarlyCancellation.subscriptionCanceled.info')}</Text>
+                    <Text>
+                        {translate('subscription.requestEarlyCancellation.subscriptionCanceled.preventFutureActivity.part1')}
+                        <TextLink onPress={() => Navigation.navigate(ROUTES.SETTINGS_WORKSPACES)}>
+                            {translate('subscription.requestEarlyCancellation.subscriptionCanceled.preventFutureActivity.link')}
+                        </TextLink>
+                        {translate('subscription.requestEarlyCancellation.subscriptionCanceled.preventFutureActivity.part2')}
+                    </Text>
+                </View>
+                <FixedFooter style={styles.ph0}>
```

Comment:
Why not move the button to a common component?

Reply:
Do you mean adding it to the return statement part? I think it's much cleaner, as this button is not rendered for every option (if that's what you meant).

Reply:
Button is what I meant.